e2e/interchaintestv8/relayer_test.go

```diff
@@ -469,6 +473,140 @@ func (s *RelayerTestSuite) ICS20TransferERC20TokenBatchedAckToEthTest(
 	}))
 }
 
+func (s *RelayerTestSuite) TestMultiPeriodClientUpdateToCosmos() {
```

Comment:
We should have some CI for this. Let's create an issue to split `e2e.yml` and link it here